src/Card/index.scss

```diff
@@ -1,12 +1,11 @@
 .nds-card {
   width: 100%;
   height: 100%;
-  box-shadow: RGB(var(--nds-dropshadow-light));
```

Comment:
do we need to add back the `box-shadow` using a standard var?

Reply:
I should have mentioned this - this var has been undefined for at least 8 months. The border defined in a declaration below this handles the card borders, so we can just remove this one